### PR TITLE
Added reference to Presidio Principles

### PR DESCRIPTION
**Pull request by VictorSyntez**
As a response to issue #26

### CODE REVIEW

File: recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 
 ## Summary
-The **Seven Laws of Identity** were [published by Kim Cameron, Architect of Identity at the Microsoft Corporation](https://www.identityblog.com/stories/2005/05/13/TheLawsOfIdentity.pdf) in 2005. This document serves to recognize this work as a collection of design principles for identity solutions.
```
Comment (vinomaster) [inline recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md:13]: In who's work and how are these presido principles relevant to laws of identity.

File: mkdocs.yml [hunk 2]
```diff
@@ -57,6 +57,7 @@ nav:
     - 'Code of Conduct': './process/code_of_conduct.md'
 - Activity Status:
     - 'Proposed': './results/proposed.md'
+    - 'Governance Stack Design Principles': './results/DP0007-governance-stack.md'
```
Comment (vinomaster) [inline mkdocs.yml:60]: This change is in error. The Yml ios not suppose to be changed. Only the phased files (proposed, accepted, approved). In this case `proposed` already has an entry for DP0007.

**Review by vinomaster — CHANGES_REQUESTED** [review]
Before merging these changes need to be incorporated.

File: recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md (hunk 1 above)
Comment (VictorSyntez) [reply recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md:13]: Frankly speaking, I didn't plan to add Presidio Principles to this document. I have followed on your proposal and added them. I would suggest to leave this document without them. Especially, since we have added Presidio Principles to Governance Stack Design Principles.

File: mkdocs.yml (hunk 2 above)
Comment (VictorSyntez) [reply mkdocs.yml:60]: Sorry, my bad. I didn't plan to edit the yml file in the repo. Have forgotten to remove before merge. It should be deleted.

File: recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md (hunk 1 above)
Comment (vinomaster) [reply recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md:13]: +1

File: mkdocs.yml (hunk 2 above)
Comment (vinomaster) [inline mkdocs.yml:60]: The PR still has a change to the YML which is not required.
